modules/ocf_puppet/files/write-otp-config

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ "$#" -ne 2 ]; then
+    echo "Usage: write-otp-config <username> <yubikey public ID>"
+    exit 1
+fi
+
+username="$1"
+yubikeyID="$2"
```

Comment:
Nit: in this repository, it seems the established convention is to use lowercase_with_underscores